config/dev.exs

```diff
@@ -2,8 +2,8 @@ import Config
 
 # Configure your database
 config :finsta, Finsta.Repo,
-  username: "postgres",
-  password: "postgres",
+  username: "newuser",
+  password: "password",
```

Comment:
Should we keep the default values in the config? I think the fewer changes the better 😅

Reply:
maybe a .env file is good for that, but for this, in the default mode I suppose yes is better 😅